File: Pemrograman1/Modul3/C/PRAK304-2310817310009-Muhamamd_Fauzan_Ahsani.c
#include <stdio.h>

int main(){
    int nilai;
    scanf("%d", &nilai);

    if(nilai>=100){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    else if(nilai < 10 && nilai > 0){
        printf("Satuan");
    }
    else if(nilai >= 11 && nilai <= 19){
        printf("Belasan");
    }
    else if(nilai == 10 || nilai >= 20 && nilai <= 99){
        printf("Puluhan");
    }
    else if (nilai == 0){
        printf("Nol");
    }
    return 0;
}

File: Pemrograman1/Modul3/Python/PRAK304-2310817310009-Muhamamd_Fauzan_Ahsani.py
nilai = int(input())

if nilai >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
elif nilai == 10 or nilai >=20 and nilai <=99:
    print("Puluhan")
elif nilai >= 11 and nilai <= 19:
    print("Belasan")
elif nilai < 10 and nilai > 0:
    print("Satuan")
elif nilai == 0:
    print("Nol")